src/main/java/io/stargate/sgv2/jsonapi/api/v1/metrics/JsonApiMetricsConfig.java

```diff
@@ -26,6 +26,18 @@ public interface JsonApiMetricsConfig {
   @WithDefault("sort.type")
   String sortType();
 
+  @NotBlank
+  @WithDefault("json.shredding.type")
+  String jsonShreddingTypeTag();
+
+  @NotBlank
+  @WithDefault("json.string.length")
+  String jsonStringLength();
+
+  @NotBlank
+  @WithDefault("json.shredding.metrics")
+  String jsonShreddingMetricsName();
```

Comment:
Maybe something like `jsonSerializationMetricsName()` with default of... `"json.shredding.serialization"`?